tox.ini

```diff
@@ -1,7 +1,21 @@
 [tox]
 envlist = py27-django{18,19,110,111}
 
+# This is needed to prevent the lms, cms, and openedx packages from getting
+# installed into site-packages where they can get imported, which is bad
+# because those won't even contain most of the source code.
+skipsdist=True
+
+# The default toxworkdir is in the source tree, but `django-admin
+# compilemessages` cannot handle encountering a .tox directory when it
+# unconditionally traverses the entire directory tree.
+toxworkdir={homedir}/.tox
+
 [testenv]
+# This ensures "-e ." is installed, so that edx-platform source code is
+# actually importable from subprocesses.
+usedevelop=True
```

Comment:
`so that a link back to the top-level directory is installed in site-packages for the top-level setup.py, making edx-platform source code importable from subprocesses.` or the like...